internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
+	Filename = "build.yml"
+	// CurrentVersion os the manifest file format. See Manifest.SDK.
+	CurrentVersion = "2"
+)
+
+// Artifact is a file copied from one path to another during the build.
+type Artifact struct {
+	Path string `yaml:"path"`
+	Dest string `yaml:"dest"`
+}
+
+// Manifest used for the declarative build system for drivers.
+type Manifest struct {
+	// SDK version. Used to track future changes to this file. Current version is '2'.
+	SDK string `yaml:"sdk"`
+
+	Native struct {
```

Comment:
What does `Native` mean in this context? Should this be `Driver`? (Again, I realize the YAML names may be fixed due to existing config files, but we control the struct layout)

Reply:
The "driver" may have 3 different meaning in this context:
- Native driver (e.g. in Java).
- Driver server (in Go).
- The driver image (both).

So answering the question, this part of the structure is about "native" part only.